packages/models-library/src/models_library/api_schemas_resource_usage_tracker/service_runs.py

```diff
@@ -43,6 +43,7 @@ class ServiceRunPage(NamedTuple):
 class OsparcCreditsAggregatedByServiceGet(BaseModel):
     osparc_credits: Decimal
     service_key: ServiceKey
+    running_time_in_hours: Decimal
```

Comment:
stupid question: why not just returning a datetime? is this not going to return a lot of 0s when someone barely uses a service?

Reply:
ah ok I answered myself later on. so this is actually a floating point value.

Reply:
correct which is aggregated over some period of time